sdk/confidentialledger/Azure.Storage.ConfidentialLedger/README.md

```diff
@@ -105,7 +103,7 @@ Response postResponse = ledgerClient.PostLedgerEntry(
     RequestContent.Create(
         new { contents = "Hello world!" }));
 
-postResponse.Headers.TryGetValue("x-ms-ccf-transaction-id", out string transactionId);
+postResponse.Headers.TryGetValue(ConfidentialLedgerConstants.TransactionId, out string transactionId);
```

Comment:
nit:  Should we consider having the word "header" in here somewhere, or do we think this can stand on its own?